hoot-core/src/main/cpp/hoot/core/io/OsmReader.cpp

```diff
@@ -595,13 +597,59 @@ bool OsmReader::startElement(const QString & /* namespaceURI */,
         {
           bool ok;
           Meters circularError = value.toDouble(&ok);
+
           if (circularError > 0 && ok)
           {
             _element->setCircularError(circularError);
           }
           else
           {
-            if (_badAccuracyCount < 10)
+            bool isBad = false;
+            hoot::Tags t1;
+            t1.setList(key, QStringList() << value);
+            //check if value contian velocity units
+            if (value.contains("km/h", Qt::CaseInsensitive) || value.contains("kph", Qt::CaseInsensitive)
+                || value.contains("kmph", Qt::CaseInsensitive) || value.contains("mph", Qt::CaseInsensitive)
+                || value.contains("knots", Qt::CaseInsensitive))
```

Comment:
The error:circular and corresponding accuracy tags should only ever store length values, not velocities.